guide/contribute/case-studies.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: guide
+title: Case Studies
+description: How to contribute case studies to the guide
+nav_order: 10
+parent: Contribute to guide
+permalink: /guide/contribute/case-studies/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/contribute/contribute-preview.jpg
+---
+
+{% include picture.html
+   image = "assets/images/guide/contribute/case-studies/CasestudiesHeader.png"
```

Comment:
[Guidelines for file names](https://bitcoin.design/guide/contribute/illustration-guidelines/#file-names).

JPEG should be smaller here than PNG ([guidelines](https://bitcoin.design/guide/contribute/illustration-guidelines/#jpg-png-or-gif)).